staging/src/kubevirt.io/api/core/v1/schema.go

```diff
@@ -732,6 +732,10 @@ type SEVPolicy struct {
 	// Defaults to false.
 	// +optional
 	EncryptedState *bool `json:"encryptedState,omitempty"`
+	// SEV-SNP is required.
+	// Defaults to false.
+	// +optional
+	SecureNestedPaging *bool `json:"secureNestedPaging,omitempty"`
```

Comment:
hi @ymjing , I happened to find that SNP has different attributes from SEV/SEV-ES, refer to docs/formatdomain.rst of libvirt source code:
     <launchSecurity type='sev-snp' authorKey='yes' vcek='no'>
       <cbitpos>47</cbitpos>
       <reducedPhysBits>1</reducedPhysBits>
       <policy>0x00030000</policy>
       <guestVisibleWorkarounds>...</guestVisibleWorkarounds>
       <idBlock>...</idBlock>
       <idAuth>...</idAuth>
       <hostData>...</hostData>
     </launchSecurity>

Besides the common attributes, e.g. cbitpos, shared with SEV/SEV-ES, it also has specialized attributes, e.g. hostData, and SNP's policy is different too, so do you think it's reasonable to introduce a SNP structure instead of taking SNP as a SEVPolicy?